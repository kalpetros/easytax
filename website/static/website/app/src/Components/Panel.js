import styles from '../../../css/src/Components/Panel.css';

import React from 'react';

class Panel extends React.Component {
    constructor(props) {
        super(props);
    }
    componentDidMount() {
    }
    render() {
	return(
	    <div className={styles.panel}>
              {this.props.children}
	    </div>
	);
    }
}

export{Panel};
