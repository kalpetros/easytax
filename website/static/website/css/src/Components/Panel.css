.panel {
    background-color: #fff;
    padding: 1rem;
    border-radius: 8px;
    box-shadow: 0 2px 2px #212121;
}
