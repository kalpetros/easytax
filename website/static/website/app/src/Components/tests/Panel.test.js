import {Panel} from '../Panel';

import renderer from 'react-test-renderer';

import React from 'react';

test('Renders a panel', () => {
  const component = renderer.create(
    <Panel>Test</Panel>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
