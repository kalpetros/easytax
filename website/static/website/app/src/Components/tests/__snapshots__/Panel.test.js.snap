// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders a panel 1`] = `
<div
  className="panel"
>
  Test
</div>
`;
